### PR TITLE
Add Screaming Frog SEO Spider.app v4.1

### PR DESCRIPTION
**Pull request by zuccs**

### CODE REVIEW

**Comment by Ngrd** [comment]
Thank you for your contribution! But it also needs `java 7+ caveat`. Example [gprojector.rb](https://github.com/caskroom/homebrew-cask/blob/325b4dd13020b197ef7a858f50581504280350f6/Casks/gprojector.rb#L12-16).

File: Casks/screaming-frog-seo-spider.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  url 'https://www.screamingfrog.co.uk/products/seo-spider/ScreamingFrogSEOSpider-#{version}.dmg'
+  name 'Screaming Frog SEO Spider'
+  name 'Screaming Frog'
+  homepage 'http://www.screamingfrog.co.uk/seo-spider/'
+  license :freemium
+
+  app 'Screaming Frog SEO Spider.app'
+
+  caveats <<-EOS.undent
+      #{token} requires Java 7+, you can install the latest Java using
```
Comment (vitorgalvao) [inline Casks/screaming-frog-seo-spider.rb]: Indentation should be two spaces, not four.

File: Casks/screaming-frog-seo-spider.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+cask :v1 => 'screaming-frog-seo-spider' do
+  version '4.1'
+  sha256 '6594f7792f04651f37dd55c2e7926649c4f27efc43e158b86f0f308ac81ee063'
+
+  url 'https://www.screamingfrog.co.uk/products/seo-spider/ScreamingFrogSEOSpider-#{version}.dmg'
```
Comment (vitorgalvao) [inline Casks/screaming-frog-seo-spider.rb]: You need to change these to double quotes, or `#{version}` won’t work.

**Comment by vitorgalvao** [comment]
Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)? @mention any of us after you do, so we get an email to check back.

**Pull request closed without merging**